pkg/interceptors/run_test.go

```diff
@@ -49,6 +49,30 @@ func ExampleRun() {
 	// interceptor 1
 }
 
+func ExampleRun_nointerceptors() {
```

Comment:
Should this one be `_no_interceptors`? Or maybe `_noInterceptors` but I don't like that one :|

Have a preference for this odd naming convention?

Reply:
Currently I've done both the underscore and no-underscore so we should probably pick one and I'll update it...

Reply:
No, I'll never look at those names again.